### DISCUSSION TITLE
Open to adding LuaCov for tracking code coverage?

### DISCUSSION THREAD

**Discussion by MunifTanjim** [nvim-neo-tree/neo-tree.nvim#418] [Internal Development]
I would like to add more tests. And for that a way to track code coverage would be nice.

Are you open to adding [LuaCov](https://github.com/keplerproject/luacov) integration with [Codecov](https://codecov.io/) for storing coverage data?

P.S.: Similar thing is being done in [nui.nvim](https://github.com/MunifTanjim/nui.nvim)

**Comment by cseickel** [comment]
> I would like to add more tests.

You had me at "I would like to add more tests." Anything that leads to more tests is a good thing. Especially if someone else is volunteering to do so.

**Reply by MunifTanjim** [reply]
I noticed that Codecov isn't posting the coverage report anymore 🤔

**Reply by cseickel** [reply]
Me too. I wonder if that's because I unsubscribed to the bot because I didn't want it in my emails... I'll take a look.

**Reply by cseickel** [reply]
I saw mention that the bot's comments can be sporadic if you don't install the Github App, so I went ahead and installed that app for the repo. We'll see on the next PR update if that works.

I also changed the target branch from v2.x to main.

**Reply by MunifTanjim** [reply]
Yep, it's back!
